champions/src/main/resources/configs/skills/skills.yml

```diff
@@ -644,11 +644,11 @@ skills:
       enabled: true
       maxlevel: 5
       cooldown: 20.0
-      radius: 3.0
+      radius: 1.5
       cooldownDecreasePerLevel: 1.0
       baseDuration: 3.0
       durationIncreasePerLevel: 1.0
-      vulnerabilityStrength: 2
+      vulnerabilityStrength: 1
```

Comment:
dont double nerf it

Reply:
Well well well I can make it vulnerability 2 again though we'll probably still have complaints

Reply:
This shit already deals 9 damage and also boosts outcoming damage for next 7 seconds,  and can be applied from 15 blocks away dude

Reply:
I've made it back to vuln 2